src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -317,6 +343,11 @@ export class HuiDialogEditCard extends LitElement {
     this._params = undefined;
     this._cardConfig = undefined;
     this._error = undefined;
+    this._documentationURL = undefined;
+  }
+
+  private _handleHelp(): void {
+    window.open(this._documentationURL, "_blank");
```

Comment:
Just make this a `<a href=${this._documentationURL} target="_blank" rel="noreferrer">`